### PR TITLE
Add moment.js to the stack.

### PR DESCRIPTION
**Pull request by johanpoirier**
Go to http://momentjs.com/ to view the full documentation.
It's a date library for parsing, validating, manipulating, and formatting
dates.
I added a 'formatDate' helper in handlebars to format date directly in the
templates. I also added 5 additional languages (fr, es, it, ru, jp).

### CODE REVIEW

File: js/libs/resthub/handlebars-helpers.js [hunk 1]
```diff
@@ -153,6 +153,39 @@ define(['handlebars', 'underscore.string'], function(Handlebars) {
...
+            }
+            else if(typeof(date) === 'string') {
+                momentDate = moment(date, (inputPattern ? inputPattern : defaultPattern));
+            }
+            else {
+                return date;
+            }
+        }
+        else {
+            return "";
```
Comment (bclozel) [inline js/libs/resthub/handlebars-helpers.js]: we talked about it this morning - is "" as a default output the sensible solution?

Comment (johanpoirier) [inline js/libs/resthub/handlebars-helpers.js]: Handlebars outputs an empty string if the value is null.
For example, `<span>{{myNullValue}}</span>` will produce `<span></span>`
So, `<span>{{formatDate myNullDate}}</span>` will also produce `<span></span>`

Comment (bclozel) [inline js/libs/resthub/handlebars-helpers.js]: Makes sense, then.
Ok!

**Comment by sdeleuze** [comment]
Sorry I broke you pullrequest, could your update/merge  your fork and push again please ?

**Comment by johanpoirier** [comment]
No problem, I'll do that as soon as I can.

**Comment by johanpoirier** [comment]
The test suite is stuck at "backbone-history: Route1", with or without my pullrequest. RestHubView issue ?

**Comment by loicfrering** [comment]
Looks like tests are passing on Travis: http://travis-ci.org/resthub/resthub-backbone-stack/builds/2788989. You see them failing locally?

**Comment by sdeleuze** [comment]
I tested it again in both Firefox and Chrome and tests are green.
Be ware to run them with http:// protocol, not file:// protocol.

In order to acheive this easily, I run a mvn jetty:run and open http://localhost:8080/tests/

**Comment by johanpoirier** [comment]
OK, tests are green with http:// protocol. It used to pass with file:// protocol last week.

**Pull request merged**